client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.module.scss

```diff
@@ -1,6 +1,17 @@
+.root {
+  min-height: 30vh;
+}
+
 .text-preview{
   max-height: 75vh;
   overflow-x: auto;
   white-space: pre-wrap;
   word-wrap: break-word;
```

Comment:
Should we be using break-word here? In the case of source code files, what is the impact?

Reply:
@owaisj , thoughts on this.

Reply:
@jchuahtacc ,  yeah, for any files with long lines (like source code files, the text is wrapped.  I feel like that could be improved (i.e don't wrap.  or if you do wrap, show line numbers to the left). I propose we open up a ticket to follow along with Design on this.  Is that okay?

Reply:
The `y` axis also is still scrollable even when the content is wrapped
<img width="781" alt="Screen Shot 2021-02-02 at 2 14 10 PM" src="https://user-images.githubusercontent.com/20326896/106657977-4f813400-6562-11eb-95ef-deebcf308d82.png">

Reply:
@rstijerina @jchuahtacc ,  i feel like if there are concerns here we should open a follow on ticket as design would probably need to be involved.  (do we wrap lines, do we not wrap lines, do we have a button to wrap/not-wrap? do we show line numbers?).

Reply:
New ticket sounds good 👍

Reply:
cool! https://jira.tacc.utexas.edu/browse/FP-901